rosboard/html/js/viewers/meta/Space3DViewer.js

```diff
@@ -57,7 +57,7 @@ class Space3DViewer extends Viewer {
     this.mvp = mat4.create();
     this.temp = mat4.create();
     
-    this.gl.captureMouse(true, true);
+    // this.gl.captureMouse(true, true);
```

Comment:
This is how you disabled the mouse event trigger?

Reply:
Yes, enabling this line will add `eventListeners` for mouse events on the rosboard canvas, which will not work with the touch events @jwdinius